Dockerfile

```diff
@@ -2,11 +2,14 @@
 FROM python:3.9
 
 # Set working directory.
-
+WORKDIR /AnshulGoyal-jtu-22-assignment
 # Expose port 8000 for your FastAPI server.
-
+EXPOSE 8000
 # Install Python requirements using Pip.
-
+#COPY ./requirements.txt /code/requirements.txt
+COPY . .
+RUN pip install -r requirements.txt
 # Copy your codebase into the Docker container.
-
+#COPY ./ /code/AnshulGoyal-jtu-22-assignment
```

Comment:
Avoid pushing commented code.
Although, copying requirements.txt and running pip install before copying the codebase is a good idea ([refer](https://stackoverflow.com/a/34399661)). Your directory structure was a bit messed up though, that's why it probably didn't work. You should try copying requirements to /AnshulGoyal-jtu-22-assignment/requirements.txt, and copy your codebase to /AnshulGoyal-jtu-22-assignment (you don't have a /code folder in your Docker container, so it doesn't make sense to reference it).